--- /usr/bin/syslinux-install_update	2018-05-30 02:01:24.000000000 +0200
+++ syslinux-install_update	2018-11-04 18:43:23.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 # Syslinux Installer / Updater Script (for BIOS only)
 # Copyright (C) 2011-2013  Matthew Gyurgyik <pyther@pyther.net>
@@ -262,7 +262,7 @@
                 exit 3
             fi
         elif [[ "$ptb" = GPT ]]; then
-            if [[ ! -e /usr/bin/sgdisk ]]; then
+            if [[ ! -e /sbin/sgdisk ]]; then
                 echo "FAILED to set attribute Legacy BIOS Bootable. sgdisk is not found - please install 'gptfdisk' package."
                 exit 3
             fi
@@ -292,8 +292,8 @@
 
         # We want to install to the root of the block device
         # If the device is a partition - ABORT!
-        dev_is_part "$disk" && \
-        { echo "ABORT! MBR installation to partition ($disk)!"; exit 4;}
+        #dev_is_part "$disk" && \
+        #{ echo "$part ABORT! MBR installation to partition ($disk)!"; exit 4;}
 
         if [[ "$ptb" = MBR ]]; then
             mbrfile="$bios_libpath/mbr.bin"
@@ -301,7 +301,8 @@
             mbrfile="$bios_libpath/gptmbr.bin"
         fi
 
-        if dd bs=440 count=1 conv=notrunc if="$mbrfile" of="$disk" &> /dev/null; then
+#        if dd bs=440 count=1 conv=notrunc if="$mbrfile" of="$disk" &> /dev/null; then
+	if dd bs=440 count=1 conv=notrunc if=/usr/lib/syslinux/mbr/gptmbr.bin of="$disk" &> /dev/null; then
             echo "Installed MBR ($mbrfile) to $disk"
         else
             echo "Error Installing MBR ($mbrfile) to $disk"
